docs/export.md

```diff
@@ -92,6 +92,8 @@ optional arguments:
   --model-path MODEL_PATH
                         Path to the folder containing model related files.
                         Signature file is required.
+  --from-type {mxnet,onnx}
```

Comment:
Can we use "model-format" and not "from-type"?
I think this better describes the argument.

Reply:
So...
--model-format {mxnet,onnx}
  Optional model source format. By default it assumes mxnet unless otherwise specified.

Reply:
See my comment above about auto-infer the format.

Reply:
Didn't we agree to change to --model-format ?